internal/process/deprovisioning/btp_operator_cleanup.go

```diff
@@ -91,7 +91,7 @@ func (s *BTPOperatorCleanupStep) Run(operation internal.Operation, log logrus.Fi
 		log.Info("skipping BTP cleanup step for real deprovisioning, not suspension")
 		return operation, 0, nil
 	}
-	if operation.ProvisioningParameters.PlanID != broker.TrialPlanID {
+	if operation.ProvisioningParameters.PlanID != broker.TrialPlanID && operation.ProvisioningParameters.PlanID != broker.FreemiumPlanID {
 		log.Info("skipping BTP cleanup step, cleanup executed only for trial plan, and this is not trial plan")
```

Comment:
Should we change the log message appropriately mentioning freemium plan as well?